src/modules/logger/logger.h

```diff
@@ -297,13 +294,7 @@ class Logger : public ModuleBase<Logger>
 
 	void write_changed_parameters(LogType type);
 
-	inline bool copy_if_updated_multi(int sub_idx, int multi_instance, void *buffer, bool try_to_subscribe);
-
-	/**
-	 * Check if a topic instance exists and subscribe to it
-	 * @return true when topic exists and subscription successful
-	 */
-	bool try_to_subscribe_topic(LoggerSubscription &sub, int multi_instance);
+	bool copy_if_updated(int sub_idx, void *buffer, bool try_to_subscribe);
```

Comment:
It seems to be worth ~ 0.4% cpu on fmu-v4 so I can leave it. In general though I'd rather change the code structure to avoid the need or even consider higher optimization levels across boards that can afford the flash.

Reply:
It won't increase flash size since it's only used once.

Reply:
Yes and it will even decrease it slightly, I just don't necessarily want to get into optimizing at that level generally.

Reply:
It's been restored if that wasn't clear.